### DISCUSSION TITLE
How do I remove highlighting?

### DISCUSSION THREAD

**Discussion by samcx** [zed-industries/zed#12201] [Feature Requests]
I find this highlighting annoying since it's highlighting everything in the query string. →

![CleanShot 2024-05-23 at 07 24 36@2x](https://github.com/zed-industries/zed/assets/28912696/a0f824b5-1ae4-493b-9e61-bfad9f677202)

Any ideas how I can disable it? I would also like to disable the highlighting for same occurrences. →

![CleanShot 2024-05-23 at 07 25 43@2x](https://github.com/zed-industries/zed/assets/28912696/7c2dedc7-45d7-44ba-aa82-25e27aa3bfb3)

Can't find the settings to disable both.

**Comment by SomeoneToIgnore** [comment]
There's

```
// How to highlight the current line in the editor.
//
// 1. Don't highlight the current line:
//    "none"
// 2. Highlight the gutter area:
//    "gutter"
// 3. Highlight the editor area:
//    "line"
// 4. Highlight the full line (default):
//    "all"
"current_line_highlight": "all",
```

to fix the first issue at least (it seems that there's no query there, just the caret placed on a wrapped line that gets highlighted due to that caret placement).

IIRC, the rest of the highlights are not possible to turn off currently.
Consider https://github.com/zed-industries/zed/pull/11710 as an inspiration for follow-up PRs 🙂

**Reply by samcx** [reply]
Thanks for the response!
